<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="characters">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="root_node">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="last_node">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_map">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Mapa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_table">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Tabla</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
